Source/Project64-core/Project64-core.vcxproj

```diff
@@ -71,13 +71,13 @@
     <ClCompile Include="N64System\Recompiler\RecompilerOps.cpp" />
     <ClCompile Include="N64System\Recompiler\RegInfo.cpp" />
     <ClCompile Include="N64System\Recompiler\SectionInfo.cpp" />
-    <ClCompile Include="N64System\Recompiler\x86CodeLog.cpp" />
+    <ClCompile Include="N64System\Recompiler\X86CodeLog.cpp" />
     <ClCompile Include="N64System\Recompiler\X86ops.cpp" />
     <ClCompile Include="N64System\SpeedLimitorClass.cpp" />
     <ClCompile Include="N64System\SystemGlobals.cpp" />
     <ClCompile Include="Plugins\AudioPlugin.cpp" />
     <ClCompile Include="Plugins\ControllerPlugin.cpp" />
-    <ClCompile Include="Plugins\GFXplugin.cpp" />
+    <ClCompile Include="Plugins\GFXPlugin.cpp" />
```

Comment:
Correct file name was `GFXplugin.cpp`, with a lowercase `'p'`.

In the future please refrain from mixing unrelated and/or wrong changes.  If a commit is named "removed spaces" it should just be about removing spaces.